### PR TITLE
Bug 1342995 - Incorrect port opened on loadbalancer host

### PR DESCRIPTION
**Pull request by abutcher**
Adds an `openshift_loadbalancer_facts` role s.t. loadbalancer facts are set prior to running other role dependencies.

https://bugzilla.redhat.com/show_bug.cgi?id=1342995

### CODE REVIEW

**Comment by abutcher** [comment]
@detiber PTAL

This was busted for < 2.0 :cry: 

```
ansible-2.1.0.0

openshift_master_api_port=443
openshift_master_console_port=443

TASK [os_firewall : Add iptables allow rules] **********************************
ok: [lb1.example.com] => (item={u'port': u'9000/tcp', u'service': u'haproxy stats'})
ok: [lb1.example.com] => (item={u'port': u'443/tcp', u'service': u'haproxy balance'})

======================================================================================

ansible-1.9.4

openshift_master_api_port=443
openshift_master_console_port=443

TASK: [os_firewall | Add iptables allow rules] ********************************
ok: [lb1.example.com] => (item={'port': '9000/tcp', 'service': 'haproxy stats'})
ok: [lb1.example.com] => (item={'port': u'8443/tcp', 'service': 'haproxy balance'})
```

File: roles/openshift_loadbalancer_facts/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+License
+-------
+
+Apache License, Version 2.0
+
+Author Information
+------------------
+
+Jason DeTiberus (jdetiber@redhat.com)
```
Comment (detiber) [inline roles/openshift_loadbalancer_facts/README.md]: Update author info?

File: roles/openshift_loadbalancer_facts/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: Jason DeTiberus
```
Comment (detiber) [inline roles/openshift_loadbalancer_facts/meta/main.yml]: same here

**Comment by detiber** [comment]
other than the author info on those two files, lgtm

**Pull request merged**